core/drivers/crypto/crypto_api/cipher/cipher.c

```diff
@@ -0,0 +1,201 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2018-2020 NXP
+ *
+ * Crypto Cipher interface implementation to enable HW driver.
+ */
+#include <assert.h>
+#include <crypto/crypto.h>
+#include <crypto/crypto_impl.h>
+#include <drvcrypt.h>
+#include <drvcrypt_cipher.h>
+#include <kernel/panic.h>
+#include <malloc.h>
+#include <utee_defines.h>
+#include <util.h>
+
+static const struct crypto_cipher_ops cipher_ops;
+
+/*
+ * Checks if input API context is correct. If not, system break.
```

Comment:
I suggest to remove this sentence. It is true only in debug mode.

Reply:
Alright